bin/compile

```diff
@@ -362,6 +369,12 @@ trap clearGitCredHelper INT TERM EXIT
 
 determineTool
 
+ver_major=0
+ver_minor=0
+ver_patch=0
+ver_special=""
+semverParseInto "$ver" ver_major ver_minor ver_patch ver_special
```

Comment:
This needs to be done after the expandVer below otherwise you get the requested version not the final one. That probably doesn't make a difference now, but it could in the future.

Reply:
Also, I think we can use something like `if semverGT 1.10 ${ver}; then .... fi` instead ?